### ISSUE TITLE
Import list of repositories from a file

### ISSUE THREAD

**Issue by den-is** [helm/helm#12542]
We are able to export list of added repositories to a file:
`helm repo ls -o json > helm_repo_ls.json`

It would be nice to have ability to import list of repositories from the same json/yaml file directly using `helm` binary, instead of using hand-made scripts.
For example:
```sh
#!/usr/bin/env bash

if ! command -v jq &> /dev/null; then
  echo "jq is not installed. Please install jq before running this script."
  exit 1
fi

if ! command -v helm &> /dev/null; then
  echo "helm is not installed. Please install helm before running this script."
  exit 1
fi

json_file="helm_repo_ls.json"

# Check if the JSON file exists
if [ ! -f "$json_file" ]; then
  echo "JSON file not found: $json_file"
  exit 1
fi

echo "Repositories to add: $(jq length helm_repo_ls.json)"

jq -c '.[]' "$json_file" | while read -r json; do
  name=$(echo "$json" | jq -r '.name')
  url=$(echo "$json" | jq -r '.url')
  if [[ -n "$name" && -n "$url" ]]; then
    echo "Adding Helm repo: $name - $url"
    helm repo add "$name" "$url" --force-update
  else
    echo "Skipping invalid entry in JSON: $json"
  fi
done

echo "Finished adding Helm repos."
```

**Comment by yxxhero** [comment]
@den-is good idea.

**Comment by gjenkins8** [comment]
I suspect (guess) you're asking, because you have automation which uses Helm and needs to import repos?

**Comment by den-is** [comment]
> I suspect (guess) you're asking, because you have automation which uses Helm and needs to import repos?

Your guess is good enough.

First of all, obviously, this feature is not needed that often.
This feature is needed when I'm setting up a new computer, or a new environment, a new home PC, or a work PC. Happens once a year or even more rarely.

But often, I'm building my "bare-metal" isolated Kubernetes Labs from scratch. Of course, I do have primary access to Labs from my main workstation that has all helm repos. But often I also have some fresh control-host that requires helm repos setup, so it is tedious to add each repo one by one when you need it.
So I export all repos from my main workstation.

Another use case. I backup my shell environment+dotfiles - environment backup includes the export of the current helm configuration too.

So cleaner out-of-the-box import sounds like a good convenience feature.

Currently, I have 44 repos. A couple of them are private, which fail during the import (because they require auth, or just are not public URLs) - and this is completely fine.

**Comment by itaispiegel** [comment]
Hey, thanks for your suggestion. I just opened a PR that implements this :)

**Comment by scottrigby** [comment]
Did you know you can already pass in a custom file path using `--repository-config` flag?

**Comment by den-is** [comment]
JFYI: Had to use the above script again today.  
For me it happens only once in 2-3 years, when I have to set up a new PC/Mac completely from scratch.  
It was a breeze.
